src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/Assertions.java

```diff
@@ -65,6 +71,57 @@ public void taskStateWasUpdatedInDatabase(String taskId, String value, Headers a
             .log();
     }
 
+    public void taskValuesWasUpdatedInDatabase(String taskId, Map<String, Matcher<?>> fieldValueMap,
```

Comment:
Maybe rename to something like `taskAttributesVerifier`?

Reply:
done